ts/src/bigone.ts

```diff
@@ -1158,6 +1160,20 @@ export default class bigone extends Exchange {
         }, market);
     }
 
+    async createMarketBuyOrderWithCost (symbol: string, cost, params = {}) {
+        /**
+         * @method
+         * @name bigone#createMarketBuyOrderWithCost
+         * @description create a market buy order by providing the symbol and cost
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {float} cost how much you want to trade in units of the quote currency
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        params['createMarketBuyOrderRequiresPrice'] = false;
+        return this.createOrder (symbol, 'market', 'buy', cost, undefined, params);
```

Comment:
@Dan-krm same the `await` is essential here otherwise we return an unwaited future in python